apps/frontend/static/js/core/state.js

```
@@ -2,10 +2,52 @@ import { tabFromLocation } from "./route.js?v=7";
+      if (typeof callback !== "function") throw new TypeError("State subscription must be a function");
+      const callbacks = subscriptions.get(key) || new Set();
+      callbacks.add(callback);
+      subscriptions.set(key, callbacks);
+      return () => callbacks.delete(callback);
+    } },
+    batch: { enumerable: false, value(callback) {
+      batching += 1;
+      try { return callback(); }
+      finally {
+        batching -= 1;
+        if (batching) return;
```

Biome (2.5.6)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)